### DISCUSSION TITLE
2024/02/25/%E5%8D%95%E7%8B%AC%E9%83%A8%E7%BD%B2Alist%E5%89%8D%E7%AB%AF%E4%BB%A5%E4%BC%98%E5%8C%96%E4%BD%93%E9%AA%8C/

### DISCUSSION THREAD

**Discussion by giscus** [bingxin666/blogmessages#5] [Announcements]
# 2024/02/25/%E5%8D%95%E7%8B%AC%E9%83%A8%E7%BD%B2Alist%E5%89%8D%E7%AB%AF%E4%BB%A5%E4%BC%98%E5%8C%96%E4%BD%93%E9%AA%8C/

这几天折腾了下部署 Alist 在服务器上，但是因为套了 Cloudflare 感觉有点慢，于是决定单独部署 Alist 前端在 Cloudflare Pages 上。

https://blog.byteloid.one/2024/02/25/%E5%8D%95%E7%8B%AC%E9%83%A8%E7%BD%B2Alist%E5%89%8D%E7%AB%AF%E4%BB%A5%E4%BC%98%E5%8C%96%E4%BD%93%E9%AA%8C/

**Comment by N501YHappy** [comment]
编译之后访问显示Request failed with status code 405

**Comment by N501YHappy** [comment]
能详细讲一下吗，谢谢了

**Comment by N501YHappy** [comment]
e,现在显示System error: TypeError: Mr is not a function

**Reply by bingxin666** [reply]
可能是你前端炸了？
抱歉我实在看不出来，因为你这个错误我真的没见过Orz...

**Reply by N501YHappy** [reply]
https://share.n501yhappy.icu/

**Reply by N501YHappy** [reply]
https://share.n501yhappy.icu
这个是我部署的
12:37:21.824	Cloning repository...
12:37:22.596	From https://github.com/N501YHappy/alist-web
12:37:22.597	 * branch            71b23a7ac19a374ecf5118c89be681f9b16e8915 -> FETCH_HEAD
12:37:22.597	
12:37:22.658	HEAD is now at 71b23a7 1
12:37:22.659	
12:37:23.385	Submodule 'solid-router' (https://github.com/alist-org/solid-router) registered for path 'solid-router'
12:37:23.385	Cloning into '/opt/buildhome/clone/solid-router'...
12:37:23.386	Submodule path 'solid-router': checked out 'f77b206e8592bb56d2d001e9488de5a54a13699b'
12:37:23.386	
12:37:23.386	Using v2 root directory strategy
12:37:23.411	Success: Finished cloning repository files
12:37:25.082	Checking for configuration in a wrangler.toml configuration file (BETA)
12:37:25.083	
12:37:25.182	No wrangler.toml file found. Continuing.
12:37:25.331	Detected the following tools from environment: pnpm@8.6.10, nodejs@18.17.1
12:37:25.669	Preparing pnpm@8.6.10 for immediate activation...
12:37:27.593	Installing project dependencies: pnpm install
12:37:28.276	 WARN  Ignoring not compatible lockfile at /opt/buildhome/repo/pnpm-lock.yaml
12:37:28.445	Progress: resolved 1, reused 0, downloaded 0, added 0
12:37:28.707	 WARN  deprecated @motionone/solid@10.14.1: Motion One for Solid is deprecated. Use solid-motionone instead https://github.com/solidjs-community/solid-motionone
12:37:29.449	Progress: resolved 46, reused 0, downloaded 30, added 0
12:37:30.449	Progress: resolved 68, reused 0, downloaded 55, added 0
12:37:31.454	Progress: resolved 108, reused 0, downloaded 91, added 0
12:37:31.987	 WARN  deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
12:37:32.455	Progress: resolved 193, reused 0, downloaded 170, added 0
12:37:32.686	 WARN  deprecated sourcemap-codec@1.4.8: Please use @jridgewell/sourcemap-codec instead
12:37:33.456	Progress: resolved 270, reused 0, downloaded 222, added 0
12:37:33.718	 WARN  deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
12:37:34.465	Progress: resolved 405, reused 0, downloaded 349, added 0
12:37:35.255	Packages: +440
12:37:35.255	++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
12:37:35.428	Packages are hard linked from the content-addressable store to the virtual store.
12:37:35.428	  Content-addressable store is at: /opt/buildhome/.pnpm-store/v3
12:37:35.428	  Virtual store is at:             node_modules/.pnpm
12:37:35.497	Progress: resolved 461, reused 0, downloaded 438, added 48
12:37:36.497	Progress: resolved 461, reused 0, downloaded 438, added 438
12:37:37.505	Progress: resolved 461, reused 0, downloaded 440, added 439
12:37:37.672	.../core-js@3.37.1/node_modules/core-js postinstall$ node -e "try{require('./postinstall')}catch(e){}"
12:37:37.755	.../core-js@3.37.1/node_modules/core-js postinstall: Done
12:37:38.139	.../esbuild@0.14.54/node_modules/esbuild postinstall$ node install.js
12:37:38.226	.../esbuild@0.14.54/node_modules/esbuild postinstall: Done
12:37:38.561	Progress: resolved 461, reused 0, downloaded 440, added 440, done
12:37:38.582	
12:37:38.582	> alist-web@0.0.0 prepare /opt/buildhome/repo
12:37:38.583	> husky install
12:37:38.583	
12:37:38.673	husky - Git hooks installed
12:37:38.691	
12:37:38.691	dependencies:
12:37:38.691	+ @github/webauthn-json 2.1.1
12:37:38.691	+ @hope-ui/solid 0.6.7
12:37:38.692	+ @monaco-editor/loader 1.4.0
12:37:38.692	+ @motionone/solid 10.14.1 (10.16.4 is available) deprecated
12:37:38.692	+ @solid-primitives/i18n 1.1.0 (2.1.1 is available)
12:37:38.692	+ @solid-primitives/keyboard 1.2.5 (1.2.8 is available)
12:37:38.692	+ @solid-primitives/storage 1.3.1 (3.5.0 is available)
12:37:38.692	+ @stitches/core 1.2.8
12:37:38.692	+ @viselect/vanilla 3.5.0 (3.5.1 is available)
12:37:38.692	+ aplayer 1.10.1
12:37:38.692	+ artplayer 5.0.9 (5.1.1 is available)
12:37:38.693	+ artplayer-plugin-danmuku 5.0.1
12:37:38.693	+ asciinema-player 3.6.3 (3.7.1 is available)
12:37:38.693	+ axios 1.0.0 (1.7.2 is available)
12:37:38.693	+ clsx 2.0.0 (2.1.1 is available)
12:37:38.693	+ copy-to-clipboard 3.3.2 (3.3.3 is available)
12:37:38.693	+ crypto-js 4.2.0
12:37:38.693	+ flv.js 1.6.2
12:37:38.693	+ hls.js 1.2.1 (1.5.8 is available)
12:37:38.693	+ just-once 2.2.0
12:37:38.694	+ libass-wasm 4.1.0
12:37:38.694	+ lightgallery 2.5.0 (2.7.2 is available)
12:37:38.694	+ mark.js 8.11.1
12:37:38.694	+ mitt 3.0.0 (3.0.1 is available)
12:37:38.694	+ rehype-katex 6.0.3 (7.0.0 is available)
12:37:38.694	+ rehype-raw 6.1.1 (7.0.0 is available)
12:37:38.694	+ remark-gfm 3.0.1 (4.0.0 is available)
12:37:38.694	+ remark-math 5.1.1 (6.0.0 is available)
12:37:38.694	+ seemly 0.3.6 (0.3.8 is available)
12:37:38.694	+ sha256 0.2.0
12:37:38.695	+ solid-contextmenu 0.0.2
12:37:38.695	+ solid-icons 1.0.1 (1.1.0 is available)
12:37:38.695	+ solid-js 1.4.8 (1.8.17 is available)
12:37:38.695	+ solid-markdown 1.2.0 (2.0.3 is available)
12:37:38.695	+ solid-transition-group 0.0.12 (0.2.3 is available)
12:37:38.695	+ streamsaver 2.0.6
12:37:38.695	+ typescript-natural-sort 0.7.2
12:37:38.695	
12:37:38.695	devDependencies:
12:37:38.696	+ @crowdin/cli 3.7.10 (3.19.4 is available)
12:37:38.696	+ @hrgui/libass-wasm-ts 1.0.3
12:37:38.696	+ @types/mark.js 8.11.8 (8.11.12 is available)
12:37:38.696	+ @types/node 18.7.5 (20.12.12 is available)
12:37:38.696	+ @types/sha256 0.2.0 (0.2.2 is available)
12:37:38.696	+ @types/streamsaver 2.0.1 (2.0.4 is available)
12:37:38.696	+ @vitejs/plugin-legacy 2.0.1 (5.4.0 is available)
12:37:38.696	+ husky 8.0.2 (9.0.11 is available)
12:37:38.696	+ lint-staged 13.0.4 (15.2.5 is available)
12:37:38.697	+ prettier 3.1.1 (3.2.5 is available)
12:37:38.697	+ rollup-plugin-copy 3.5.0
12:37:38.697	+ terser 5.14.2 (5.31.0 is available)
12:37:38.697	+ typescript 4.7.4 (5.4.5 is available)
12:37:38.697	+ vite 3.0.8 (5.2.11 is available)
12:37:38.697	+ vite-plugin-dynamic-base 0.4.4 (1.1.0 is available)
12:37:38.697	+ vite-plugin-solid 2.3.0 (2.10.2 is available)
12:37:38.697	
12:37:38.698	 WARN  Issues with peer dependencies found
12:37:38.698	.
12:37:38.698	├─┬ @hope-ui/solid 0.6.7
12:37:38.698	│ └── ✕ unmet peer solid-transition-group@^0.0.10: found 0.0.12
12:37:38.698	├─┬ solid-contextmenu 0.0.2
12:37:38.699	│ └── ✕ unmet peer solid-transition-group@^0.0.10: found 0.0.12
12:37:38.699	├─┬ @motionone/solid 10.14.1
12:37:38.699	│ └─┬ @solid-primitives/props 2.2.2
12:37:38.699	│   └─┬ @solid-primitives/utils 3.1.0
12:37:38.699	│     └── ✕ unmet peer solid-js@^1.5.0: found 1.4.8
12:37:38.699	└─┬ @solid-primitives/keyboard 1.2.5
12:37:38.699	  ├── ✕ unmet peer solid-js@^1.6.12: found 1.4.8
12:37:38.699	  ├─┬ @solid-primitives/event-listener 2.3.3
12:37:38.700	  │ ├── ✕ unmet peer solid-js@^1.6.12: found 1.4.8
12:37:38.700	  │ └─┬ @solid-primitives/utils 6.2.3
12:37:38.700	  │   └── ✕ unmet peer solid-js@^1.6.12: found 1.4.8
12:37:38.700	  └─┬ @solid-primitives/rootless 1.4.5
12:37:38.700	    └── ✕ unmet peer solid-js@^1.6.12: found 1.4.8
12:37:38.700	
12:37:38.700	Done in 10.7s
12:37:38.738	Executing user command: pnpm install && pnpm build
12:37:39.476	Lockfile is up to date, resolution step is skipped
12:37:39.527	Already up to date
12:37:39.844	
12:37:39.862	
12:37:39.862	> alist-web@0.0.0 prepare /opt/buildhome/repo
12:37:39.862	> husky install
12:37:39.863	
12:37:39.950	husky - Git hooks installed
12:37:39.958	Done in 878ms
12:37:40.611	
12:37:40.611	> alist-web@0.0.0 build /opt/buildhome/repo
12:37:40.611	> vite build
12:37:40.611	
12:37:41.142	vite v3.0.8 building for production...
12:37:41.191	transforming...
12:37:52.313	✓ 739 modules transformed.
12:38:02.445	rendering chunks...
12:38:17.979	dist/assets/useTitle-legacy.cee7b002.js                         0.59 KiB / gzip: 0.38 KiB
12:38:17.979	dist/assets/index-legacy.db88599c.js                            3.18 KiB / gzip: 1.65 KiB
12:38:17.979	dist/assets/index-legacy.fcc5f40f.js                            8.78 KiB / gzip: 3.09 KiB
12:38:17.979	dist/assets/FolderTree-legacy.578ae7ef.js                       3.77 KiB / gzip: 1.57 KiB
12:38:17.981	dist/assets/Layout-legacy.408cd3db.js                           36.18 KiB / gzip: 11.05 KiB
12:38:17.981	dist/assets/index-legacy.9f32fa1a.js                            7.52 KiB / gzip: 3.02 KiB
12:38:17.981	dist/assets/index-legacy.094ebc42.js                            9.43 KiB / gzip: 3.16 KiB
12:38:17.982	dist/assets/index-legacy.86a91193.js                            3.87 KiB / gzip: 0.65 KiB
12:38:17.982	dist/assets/webauthn-json.browser-ponyfill-legacy.8ddd819a.js   2.29 KiB / gzip: 0.96 KiB
12:38:17.982	dist/assets/index-legacy.272df1c7.js                            0.36 KiB / gzip: 0.26 KiB
12:38:17.982	dist/assets/Upload-legacy.eb2026bf.js                           6.08 KiB / gzip: 2.43 KiB
12:38:17.982	dist/assets/icon-legacy.cb21c0a5.js                             5.02 KiB / gzip: 2.44 KiB
12:38:17.982	dist/assets/entry-legacy.2b2db82f.js                            35.69 KiB / gzip: 10.13 KiB
12:38:17.982	dist/assets/Paginator-legacy.ec6fae23.js                        2.01 KiB / gzip: 0.75 KiB
12:38:17.982	dist/assets/_commonjs-dynamic-modules-legacy.cf19cda3.js        0.30 KiB / gzip: 0.23 KiB
12:38:17.983	dist/assets/File-legacy.b46c2d01.js                             5.12 KiB / gzip: 2.01 KiB
12:38:17.983	dist/assets/Password-legacy.a208b351.js                         1.28 KiB / gzip: 0.67 KiB
12:38:17.983	dist/assets/Common-legacy.fd20bab8.js                           1.45 KiB / gzip: 0.77 KiB
12:38:17.983	dist/assets/PackageDownload-legacy.7425e502.js                  9.05 KiB / gzip: 3.84 KiB
12:38:17.983	dist/assets/SettingItem-legacy.212b7235.js                      2.41 KiB / gzip: 0.92 KiB
12:38:17.983	dist/assets/ResponsiveGrid-legacy.f02591cc.js                   0.35 KiB / gzip: 0.28 KiB
12:38:17.983	dist/assets/item_type-legacy.d1f1c701.js                        0.18 KiB / gzip: 0.16 KiB
12:38:17.983	dist/assets/S3-legacy.c8c56511.js                               4.01 KiB / gzip: 1.57 KiB
12:38:17.983	dist/assets/Profile-legacy.55fa59bb.js                          5.90 KiB / gzip: 2.13 KiB
12:38:17.984	dist/assets/Other-legacy.8f13f3b3.js                            2.69 KiB / gzip: 0.96 KiB
12:38:17.984	dist/assets/offline_download-legacy.6b91d143.js                 0.52 KiB / gzip: 0.33 KiB
12:38:17.984	dist/assets/Upload-legacy.13d02c12.js                           0.36 KiB / gzip: 0.25 KiB
12:38:17.984	dist/assets/Tasks-legacy.5c4fed2b.js                            4.20 KiB / gzip: 1.50 KiB
12:38:17.984	dist/assets/Copy-legacy.beddff48.js                             0.36 KiB / gzip: 0.26 KiB
12:38:17.984	dist/assets/DeletePopover-legacy.0f0be997.js                    0.73 KiB / gzip: 0.37 KiB
12:38:17.984	dist/assets/Users-legacy.18c20c5f.js                            2.76 KiB / gzip: 1.11 KiB
12:38:17.984	dist/assets/Storages-legacy.165346af.js                         4.84 KiB / gzip: 1.71 KiB
12:38:17.985	dist/assets/Wether-legacy.13204b9b.js                           0.29 KiB / gzip: 0.22 KiB
12:38:17.985	dist/assets/Metas-legacy.bc857b8b.js                            1.85 KiB / gzip: 0.83 KiB
12:38:17.985	dist/assets/backup-restore-legacy.a3eb0f01.js                   5.51 KiB / gzip: 2.22 KiB
12:38:17.985	dist/assets/About-legacy.ba3fab8a.js                            0.47 KiB / gzip: 0.33 KiB
12:38:17.985	dist/assets/indexes-legacy.db3b56c9.js                          4.92 KiB / gzip: 1.76 KiB
12:38:17.985	dist/assets/AddOrEdit-legacy.29069a71.js                        4.98 KiB / gzip: 1.78 KiB
12:38:17.985	dist/assets/AddOrEdit-legacy.5c825434.js                        2.92 KiB / gzip: 1.12 KiB
12:38:17.986	dist/assets/AddOrEdit-legacy.7bc94ad0.js                        2.87 KiB / gzip: 1.16 KiB
12:38:17.986	dist/assets/2fa-legacy.6151a4af.js                              1.31 KiB / gzip: 0.71 KiB
12:38:17.986	dist/assets/List-legacy.cff61629.js                             4.33 KiB / gzip: 1.84 KiB
12:38:17.986	dist/assets/Messenger-legacy.d444469a.js                        1.41 KiB / gzip: 0.72 KiB
12:38:17.986	dist/assets/index-legacy.fdcf1249.js                            2.17 KiB / gzip: 1.02 KiB
12:38:17.986	dist/assets/GridItem-legacy.0309cb6b.js                         2.76 KiB / gzip: 1.32 KiB
12:38:17.986	dist/assets/helper-legacy.12598ff4.js                           14.23 KiB / gzip: 5.06 KiB
12:38:17.987	dist/assets/Grid-legacy.be0c571f.js                             1.16 KiB / gzip: 0.61 KiB
12:38:17.987	dist/assets/ImageWithError-legacy.8eef448a.js                   0.32 KiB / gzip: 0.24 KiB
12:38:17.987	dist/assets/html-legacy.df5bffb5.js                             0.65 KiB / gzip: 0.39 KiB
12:38:17.987	dist/assets/Images-legacy.4fb4f260.js                           3.42 KiB / gzip: 1.55 KiB
12:38:17.987	dist/assets/markdown-legacy.5ae80dfb.js                         0.35 KiB / gzip: 0.25 KiB
12:38:17.987	dist/assets/markdown_with_word_wrap-legacy.95793ded.js          0.32 KiB / gzip: 0.24 KiB
12:38:17.987	dist/assets/aliyun_video-legacy.488fc1d8.js                     6.63 KiB / gzip: 2.78 KiB
12:38:17.987	dist/assets/url-legacy.2e443f90.js                              0.79 KiB / gzip: 0.43 KiB
12:38:17.987	dist/assets/image-legacy.3e72ee34.js                            0.42 KiB / gzip: 0.30 KiB
12:38:17.988	dist/assets/ipa-legacy.98e00346.js                              1.11 KiB / gzip: 0.59 KiB
12:38:17.988	dist/assets/text-editor-legacy.605834e9.js                      9.53 KiB / gzip: 3.54 KiB
12:38:17.988	dist/assets/plist-legacy.e1da2962.js                            0.71 KiB / gzip: 0.42 KiB
12:38:17.988	dist/assets/aliyun_office-legacy.6d81abe2.js                    0.57 KiB / gzip: 0.39 KiB
12:38:17.988	dist/assets/index-legacy.98f41633.js                            65.38 KiB / gzip: 25.31 KiB
12:38:17.988	dist/assets/audio-legacy.70fcad66.js                            72.54 KiB / gzip: 16.71 KiB
12:38:17.988	dist/assets/polyfills-legacy.e272c637.js                        47.90 KiB / gzip: 18.96 KiB
12:38:17.988	dist/assets/video-legacy.1709b01e.js                            164.91 KiB / gzip: 39.32 KiB
12:38:17.988	dist/assets/video_box-legacy.db07a323.js                        162.20 KiB / gzip: 41.31 KiB
12:38:17.989	dist/assets/Folder-legacy.088a18a4.js                           150.61 KiB / gzip: 40.98 KiB
12:38:17.992	dist/assets/asciinema-legacy.b70eec4c.js                        199.06 KiB / gzip: 69.07 KiB
12:38:17.996	dist/assets/hls-legacy.79fdc595.js                              437.52 KiB / gzip: 120.87 KiB
12:38:18.038	dist/assets/index-legacy.116fb1ae.js                            1144.67 KiB / gzip: 336.58 KiB
12:38:18.039	
12:38:18.040	(!) Some chunks are larger than 500 KiB after minification. Consider:
12:38:18.040	- Using dynamic import() to code-split the application
12:38:18.040	- Use build.rollupOptions.output.manualChunks to improve chunking: https://rollupjs.org/guide/en/#outputmanualchunks
12:38:18.040	- Adjust chunk size limit for this warning via build.chunkSizeWarningLimit.
12:38:19.002	dist/assets/loading.298ad3ff.gif                         4.08 KiB
12:38:19.002	dist/assets/lg.22b72ba5.ttf                              4.64 KiB
12:38:19.003	dist/assets/lg.fefc5c0d.woff                             4.72 KiB
12:38:19.003	dist/assets/lg.f2fe1c00.svg                              12.53 KiB
12:38:19.003	dist/assets/subtitles-octopus-worker.f95f2186.js         390.32 KiB
12:38:19.003	dist/assets/TimesNewRoman.d9fb32eb.ttf                   814.69 KiB
12:38:19.003	dist/assets/subtitles-octopus-worker.62892886.wasm       2251.80 KiB
12:38:19.003	dist/assets/SourceHanSansCN-Bold.b5627435.woff2          4885.68 KiB
12:38:19.006	dist/index.html                                          3.87 KiB
12:38:19.017	dist/assets/useTitle.bb694357.js                         0.49 KiB / gzip: 0.32 KiB
12:38:19.017	dist/assets/index.de95fb93.js                            3.24 KiB / gzip: 1.67 KiB
12:38:19.018	dist/assets/FolderTree.949bc1bb.js                       3.88 KiB / gzip: 1.60 KiB
12:38:19.018	dist/assets/index.02c9057f.js                            8.80 KiB / gzip: 3.10 KiB
12:38:19.019	dist/assets/index.f97b8c2b.js                            3.80 KiB / gzip: 0.60 KiB
12:38:19.020	dist/assets/index.6a2ebb94.js                            12.65 KiB / gzip: 3.40 KiB
12:38:19.020	dist/assets/webauthn-json.browser-ponyfill.1c672167.js   2.31 KiB / gzip: 0.96 KiB
12:38:19.021	dist/assets/index.4d83c563.js                            0.26 KiB / gzip: 0.20 KiB
12:38:19.021	dist/assets/index.2eed1b1c.js                            7.48 KiB / gzip: 2.97 KiB
12:38:19.021	dist/assets/Upload.c252bf8a.js                           6.13 KiB / gzip: 2.54 KiB
12:38:19.022	dist/assets/icon.da4dbe0d.js                             4.89 KiB / gzip: 2.37 KiB
12:38:19.022	dist/assets/Paginator.fabd14d9.js                        1.96 KiB / gzip: 0.71 KiB
12:38:19.022	dist/assets/Layout.46bd499b.js                           36.72 KiB / gzip: 11.09 KiB
12:38:19.023	dist/assets/_commonjs-dynamic-modules.30ae7933.js        0.23 KiB / gzip: 0.19 KiB
12:38:19.023	dist/assets/Password.e91462b3.js                         1.19 KiB / gzip: 0.62 KiB
12:38:19.023	dist/assets/File.47f236de.js                             7.36 KiB / gzip: 2.19 KiB
12:38:19.023	dist/assets/Common.cf39d86a.js                           1.32 KiB / gzip: 0.69 KiB
12:38:19.023	dist/assets/entry.5b80f16d.js                            37.17 KiB / gzip: 11.26 KiB
12:38:19.023	dist/assets/PackageDownload.585974c3.js                  9.06 KiB / gzip: 3.87 KiB
12:38:19.023	dist/assets/item_type.be442da4.js                        0.12 KiB / gzip: 0.11 KiB
12:38:19.023	dist/assets/ResponsiveGrid.fe22e4d9.js                   0.28 KiB / gzip: 0.24 KiB
12:38:19.024	dist/assets/SettingItem.463895c9.js                      2.51 KiB / gzip: 0.89 KiB
12:38:19.024	dist/assets/S3.35a80e6b.js                               3.84 KiB / gzip: 1.50 KiB
12:38:19.024	dist/assets/offline_download.19ce4c49.js                 0.42 KiB / gzip: 0.28 KiB
12:38:19.024	dist/assets/Profile.b0642b4d.js                          5.77 KiB / gzip: 2.04 KiB
12:38:19.024	dist/assets/Other.07b2f2d5.js                            2.64 KiB / gzip: 0.94 KiB
12:38:19.024	dist/assets/Upload.b9dbf896.js                           0.26 KiB / gzip: 0.19 KiB
12:38:19.024	dist/assets/Tasks.5068f4a5.js                            4.12 KiB / gzip: 1.47 KiB
12:38:19.024	dist/assets/Copy.5b7f15e1.js                             0.27 KiB / gzip: 0.21 KiB
12:38:19.024	dist/assets/DeletePopover.ea9f1d0a.js                    0.65 KiB / gzip: 0.32 KiB
12:38:19.024	dist/assets/Wether.4498e574.js                           0.21 KiB / gzip: 0.18 KiB
12:38:19.025	dist/assets/Users.46fbfa01.js                            2.65 KiB / gzip: 1.04 KiB
12:38:19.025	dist/assets/Storages.e3d9b91d.js                         4.73 KiB / gzip: 1.61 KiB
12:38:19.025	dist/assets/Metas.4171faed.js                            1.74 KiB / gzip: 0.75 KiB
12:38:19.025	dist/assets/indexes.7073f17f.js                          4.87 KiB / gzip: 1.71 KiB
12:38:19.025	dist/assets/About.d5fc1340.js                            0.38 KiB / gzip: 0.27 KiB
12:38:19.025	dist/assets/AddOrEdit.c75d891f.js                        2.82 KiB / gzip: 1.05 KiB
12:38:19.025	dist/assets/backup-restore.c527ce03.js                   5.44 KiB / gzip: 2.22 KiB
12:38:19.025	dist/assets/AddOrEdit.97eb4c89.js                        5.11 KiB / gzip: 1.76 KiB
12:38:19.025	dist/assets/Messenger.63c280e8.js                        1.31 KiB / gzip: 0.67 KiB
12:38:19.025	dist/assets/AddOrEdit.eaf2a769.js                        2.75 KiB / gzip: 1.09 KiB
12:38:19.025	dist/assets/List.37e23bf5.js                             4.15 KiB / gzip: 1.76 KiB
12:38:19.026	dist/assets/index.7dacaf83.js                            2.10 KiB / gzip: 1.00 KiB
12:38:19.026	dist/assets/Grid.60a0f52d.js                             0.97 KiB / gzip: 0.56 KiB
12:38:19.026	dist/assets/ImageWithError.426a42ce.js                   0.24 KiB / gzip: 0.19 KiB
12:38:19.026	dist/assets/GridItem.5522debc.js                         2.66 KiB / gzip: 1.25 KiB
12:38:19.026	dist/assets/html.22b66d91.js                             0.59 KiB / gzip: 0.36 KiB
12:38:19.026	dist/assets/2fa.2cb37011.js                              1.28 KiB / gzip: 0.67 KiB
12:38:19.026	dist/assets/Images.3aa5b663.js                           3.25 KiB / gzip: 1.48 KiB
12:38:19.026	dist/assets/markdown.d01dc958.js                         0.30 KiB / gzip: 0.22 KiB
12:38:19.026	dist/assets/markdown_with_word_wrap.333d75af.js          0.27 KiB / gzip: 0.21 KiB
12:38:19.027	dist/assets/url.d9d1bb8c.js                              0.66 KiB / gzip: 0.39 KiB
12:38:19.027	dist/assets/aliyun_video.727e78a4.js                     6.63 KiB / gzip: 2.78 KiB
12:38:19.027	dist/assets/image.2e303b93.js                            0.34 KiB / gzip: 0.25 KiB
12:38:19.027	dist/assets/helper.81a203b8.js                           14.26 KiB / gzip: 5.15 KiB
12:38:19.027	dist/assets/text-editor.58d3806b.js                      9.74 KiB / gzip: 3.64 KiB
12:38:19.027	dist/assets/ipa.836e80ed.js                              0.97 KiB / gzip: 0.53 KiB
12:38:19.027	dist/assets/plist.0dd4f3b4.js                            0.58 KiB / gzip: 0.36 KiB
12:38:19.027	dist/assets/aliyun_office.3f71cc0e.js                    0.49 KiB / gzip: 0.35 KiB
12:38:19.028	dist/assets/audio.5075d753.js                            58.88 KiB / gzip: 14.76 KiB
12:38:19.028	dist/assets/audio.e5b5af14.css                           13.23 KiB / gzip: 2.63 KiB
12:38:19.028	dist/assets/index.15d8d646.css                           19.40 KiB / gzip: 4.11 KiB
12:38:19.028	dist/assets/Folder.b511ba9e.css                          35.83 KiB / gzip: 8.56 KiB
12:38:19.028	dist/assets/asciinema.53412307.css                       43.92 KiB / gzip: 6.43 KiB
12:38:19.028	dist/assets/index.ced13958.js                            68.04 KiB / gzip: 25.71 KiB
12:38:19.028	dist/assets/Folder.11404768.js                           125.14 KiB / gzip: 33.06 KiB
12:38:19.028	dist/assets/video_box.0d206eef.js                        167.66 KiB / gzip: 42.18 KiB
12:38:19.029	dist/assets/video.c9539b0d.js                            158.07 KiB / gzip: 40.16 KiB
12:38:19.033	dist/assets/asciinema.2e973d13.js                        156.63 KiB / gzip: 62.96 KiB
12:38:19.040	dist/assets/hls.7efce7e0.js                              414.28 KiB / gzip: 123.40 KiB
12:38:19.139	dist/assets/index.96025237.js                            1145.77 KiB / gzip: 343.48 KiB
12:38:19.140	
12:38:19.141	(!) Some chunks are larger than 500 KiB after minification. Consider:
12:38:19.141	- Using dynamic import() to code-split the application
12:38:19.141	- Use build.rollupOptions.output.manualChunks to improve chunking: https://rollupjs.org/guide/en/#outputmanualchunks
12:38:19.141	- Adjust chunk size limit for this warning via build.chunkSizeWarningLimit.
12:38:19.726	Finished
12:38:19.727	Note: No functions dir at /functions found. Skipping.
12:38:19.727	Validating asset output directory
12:38:20.564	Deploying your site to Cloudflare's global network...
12:38:23.425	Uploading... (161/161)
12:38:23.426	✨ Success! Uploaded 0 files (161 already uploaded) (0.37 sec)
12:38:23.426	
12:38:23.703	✨ Upload complete!
12:38:25.900	Success: Assets published!
12:38:26.804	Success: Your site was deployed!

我没去做汉化

**Comment by kufx** [comment]
大佬，这个用vercel部署好前端就可以使用了吗？

**Comment by YeisuQwQ** [comment]
大佬，根据GitHub上的readme访问http://localhost:5173/的时候一直在加载怎么办？

**Reply by bingxin666** [reply]
可能是跨域炸了？
